userProfile/src/Components/Form.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { Button, Input, Gapped, Select, Modal } from '@skbkontur/react-ui';
+
+const cities: string[] = ['Екатеринбург', 'Н.Тагил', 'Москва', 'Санкт-Петербург', 'Казань'];
+
+interface FormData {
+    name: string;
+    surname: string;
+    city: string | undefined;
+}
+
+const saveData: FormData = {
+    name: '',
+    surname: '',
+    city: ''
+};
+
+const defaultData: FormData = {
+    name: '',
+    surname: '',
+    city: cities[0]
+};
+
+const Form = () => {
+    const [name, setName] = useState(defaultData.name);
+    const [surname, setSurname] = useState(defaultData.surname);
+    const [city, setCity] = useState(defaultData.city);
+
+    const [person, setPerson] = useState({
+        name,
+        surname,
+        city
+    });
+
+    const [opened, setOpened] = useState(false);
+
+    const handlerField = (field: keyof FormData, value: string) => {
+        switch (field) {
+            case 'name':
+                setName(value);
+                break;
+            case 'surname':
+                setSurname(value);
+                break;
+            case 'city':
+                setCity(value);
+                break;
+            default:
+                throw new Error();
+        }
+    };
+
+    const submitForm = () => {
+        setPerson({ name: name, surname: surname, city: city });
+        setOpened(true);
+    };
+
+    const renderForm = () => {
+        return (
+            <form className="form">
+                <Gapped gap={15} vertical>
+                    <label>
+                        <div className="label">Имя</div>
+                        <Input
+                            placeholder="Введите имя пользователя"
+                            value={name}
+                            onChange={e => handlerField('name', e.target.value)}
+                        />
+                    </label>
+                    <label>
+                        <div className="label">Фамилия</div>
+                        <Input
+                            placeholder="Введите имя пользователя"
+                            value={surname}
+                            onChange={e => handlerField('surname', e.target.value)}
+                        />
+                    </label>
+                    <label>
+                        <div className="label">Город</div>
+                        <Select<string>
+                            placeholder="Выберите город"
+                            items={cities}
+                            value={city}
+                            onValueChange={e => handlerField('city', e)}
```

Comment:
В стрелочных функциях часто используется одна буква в качестве названия переменной, обычно это первая буква полного названия. `onValueChange` является функцией, принимающей единственный аргумент `value`, из документации:
```
onValueChange?: (value: TValue) => void;
```
А значит чтобы не возникло путаницы, и никто не подумал, что здесь фигурирует эвент, стоит называть переменную `value` или сокращенно `v`
```suggestion
                            onValueChange={v => handlerField('city', v)}
```